src/fragments/lib/troubleshooting/common/upgrading.mdx

```diff
@@ -151,7 +151,7 @@ amplify upgrade
 amplify push
 ```
 
-This will generate a new configuration file called `amplifyconfiguration.json`
+This will generate a new configuration file called `amplifyconfiguration.json` (You may need to use amplify pull if there are no changes in your environment).
```

Comment:
```suggestion
This will generate a new configuration file called `amplifyconfiguration.json` (You may need to use `amplify pull` if there are no changes in your environment).
```